zkevm-circuits/src/evm_circuit/util/constraint_builder.rs

```diff
@@ -1053,8 +1053,8 @@ impl<'a, F: FieldExt> ConstraintBuilder<'a, F> {
             RwTableTag::TxLog,
             [
                 tx_id,
-                index + (1u64 << 8).expr() * log_id,
-                tag.expr(),
+                index + (1u64 << 32).expr() * field_tag.expr() + (1u64 << 48).expr() * log_id,
```

Comment:
one minor question is why reserve 16 bits for the `field_tag`. It seems only 2 bits would be necessary in the case of `TxLog`.

Reply:
It's not required, but aligning the fields to 16 bit limbs means that the field tag in a separate advice column from the index and log_id.

Reply:
yeah, 16 bits fit into one limb of state circuit limb column.